nbs/tutorials/01_quickdraw.py

```diff
@@ -0,0 +1,28 @@
+from fasthtml.common import *
+from datetime import datetime
+
+def render(room):
+    return Li(A(room.name, href=f"/rooms/{room.id}"))
+
+app,rt,rooms,Room = fast_app('data/drawapp.db', hdrs=(picolink,), render=render, id=int, name=str, created_at=str, pk='id')
+
+@rt("/")
+def get():
+    create_room = Form(Input(id="room-name", name="name", placeholder="New Room Name"),
+                       Button("Create Room"),
+                       hx_post="/rooms", hx_target="#rooms-list", hx_swap="afterbegin")
+    rooms_list = Ul(*rooms(order_by='id DESC'), id='rooms-list')
+    return Titled("QuickDraw", create_room, rooms_list)
+
+@rt("/rooms")
+async def post(room:Room):
+    room.created_at = datetime.now().isoformat()
+    new_room = rooms.insert(room)
```

Comment:
```suggestion
    return rooms.insert(room)
```

The render function should handle this.